cli/pack_command.go

```diff
@@ -0,0 +1,110 @@
+package cli
+
+import (
+	"github.com/heroku/color"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/buildpacks/pack"
+	"github.com/buildpacks/pack/buildpackage"
+	"github.com/buildpacks/pack/cmd"
+	"github.com/buildpacks/pack/internal/commands"
+	"github.com/buildpacks/pack/internal/config"
+	clilogger "github.com/buildpacks/pack/internal/logging"
+	"github.com/buildpacks/pack/logging"
+)
+
+var (
+	packClient pack.Client
```

Comment:
This might be carried over from the move to this new file but what's the purpose of setting the package level variable? It doesn't appear to be used.

Reply:
Hmm. I took that from the existing code, but it looks like it was created originally to allow for initializing it in PreRun. I can try without doing that though.

Reply:
It was added in here, when we first created pack.Client: https://github.com/buildpacks/pack/commit/237719e6220f4ced26a1670ac3bfb1a944bee8c7#diff-d9d0491c5f9a7fee85ac4da6dc4f2058. It works without it though.